app/helpers/placements/support/organisations_helper.rb

```diff
@@ -11,6 +11,19 @@ def no_results(search = "", filters = [])
     end
   end
 
+  def organisation_url(organisation)
+    case organisation.class.name
+    when "Placements::School"
+      placements_support_school_path(organisation)
+    when "School"
+      placements_support_school_path(organisation)
+    when "Placements::Provider"
+      placements_support_provider_path(organisation)
+    when "Provider"
+      placements_support_provider_path(organisation)
+    end
+  end
```

Comment:
This supports inheritence:

```suggestion
  def organisation_url(organisation)
    case organisation
    when School
      placements_support_school_path(organisation)
    when Provider
      placements_support_provider_path(organisation)
    end
  end
```